vita3k/mem/src/mem.cpp

```diff
@@ -43,7 +43,8 @@ static void delete_memory(uint8_t *memory) {
 }
 
 static void alloc_inner(MemState &state, Address address, size_t page_count, Allocated::iterator block, const char *name) {
-    uint8_t *const memory = &state.memory[address];
+    //uint8_t *const memory = &state.memory[address];
+    uint8_t * memory = &state.memory[address];
```

Comment:
I'm not sure if we need this either. I was okay with the header change btw!

Reply:
Yep, this should not be changed. :D

Reply:
Hey, May I ask what to put in commit message in this case. I don't know how to name this commit message, XD. Any suggestion?

Reply:
OK, just make a third commit.

Reply:
Awesome. We usually name our commits like this `module: what commit does`. I would've probably named the whole thing something like `mem: fix broken protects on macos`.

We'll probably squash all 3 commits before merging if that's okay :)

Reply:
OK, that sounds great.